File: src/WebClient/WareHouse/src/app/entity/SerialWareHouse.ts
import { BaseEntity } from "./BaseEntity";
import { Inward } from "./Inward";
import { InwardDetail } from "./InwardDetail";
import { OutwardDetail } from "./OutwardDetail";
import { WareHouseItem } from "./WareHouseItem";

export interface SerialWareHouse extends BaseEntity {
    itemId: string;
    serial: string;
    inwardDetailId: string;
    outwardDetailId: string;
    isOver: boolean;
    inwardDetail: InwardDetail;
    item: WareHouseItem;
    outwardDetail: OutwardDetail;
}

File: src/Services/WareHouse/WareHouse.Domain/Entity/SerialWareHouse.cs
using System;
using System.Collections.Generic;
using Share.BaseCore;

#nullable disable

namespace WareHouse.Domain.Entity
{
    public partial class SerialWareHouse:BaseEntity
    {
        public string ItemId { get; set; }
        public string Serial { get; set; }
        public string InwardDetailId { get; set; }
        public string OutwardDetailId { get; set; }
        public bool IsOver { get; set; }

        public virtual InwardDetail InwardDetail { get; set; }
        public virtual WareHouseItem Item { get; set; }
        public virtual OutwardDetail OutwardDetail { get; set; }
    }
}
